### PR TITLE
Update changelog for v5.0 release

### PR DESCRIPTION
**Pull request by nick-mobilecoin**
### Future Work

[Soundtrack of this PR]()

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,14 +8,264 @@ The crates in this repository do not adhere to [Semantic Versioning](https://sem
 
 ## [5.0.0]
 
+### Added
+
+- Average slam rate to json slam report ([#3080])
```
Comment (samdealy) [inline CHANGELOG.md:13]: Can you change `Average` to some verb that describes the change to the slam rate?

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,14 +8,264 @@ The crates in this repository do not adhere to [Semantic Versioning](https://sem
...
+- Provide build time error messages for invalid SGX_MODE and IAS_MODE environment variables ([#3164])
+- `mc-attest-verifier-config` crate to handle multiple enclave measurements and hardening advisories ([#3148])
+- Logic in consensus to track and drop clients that exceed a number of failed tx proposals ([#3208], [#3155], [#3156])
+- Passing extra GRPC services to `AdminServer` ([#3242])
```
Comment (samdealy) [inline CHANGELOG.md:17]:
```suggestion
- Pass extra GRPC services to `AdminServer` ([#3242])
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,14 +8,264 @@ The crates in this repository do not adhere to [Semantic Versioning](https://sem
...
+- [MCIP 61]: Defragmentation Memos ([#3170])
```
Comment (samdealy) [inline CHANGELOG.md:18]:
```suggestion
- [MCIP 61]: Add Defragmentation Memos ([#3170])
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -8,14 +8,264 @@ The crates in this repository do not adhere to [Semantic Versioning](https://sem
...
+- Capability of b58-decoder to print public address hashes ([#3283])
```
Comment (samdealy) [inline CHANGELOG.md:19]:
```suggestion
- Enable b58-decoder to print public address hashes ([#3283])
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -8,14 +8,264 @@ The crates in this repository do not adhere to [Semantic Versioning](https://sem
...
+- Fog view router to support sharding the fog storage. Provides a streaming and unary GRPC api. ([#3297])
```
Comment (samdealy) [inline CHANGELOG.md:20]:
```suggestion
- Fog view router supports sharding the fog storage. Provides a streaming and unary GRPC api. ([#3297])
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -8,14 +8,264 @@ The crates in this repository do not adhere to [Semantic Versioning](https://sem
...
+- Fog ledger router to support sharding the key image storage. Provides a streaming and unary GRPC api. ([#3312], ([#3331])
```
Comment (samdealy) [inline CHANGELOG.md:21]:
```suggestion
- Fog ledger router supports sharding the key image storage. Provides a streaming and unary GRPC api. ([#3312], ([#3331])
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -8,14 +8,264 @@ The crates in this repository do not adhere to [Semantic Versioning](https://sem
 
 ## [5.0.0]
 
+### Added
+
+- Average slam rate to json slam report ([#3080])
+- Provide build time error messages for invalid SGX_MODE and IAS_MODE environment variables ([#3164])
+- `mc-attest-verifier-config` crate to handle multiple enclave measurements and hardening advisories ([#3148])
+- Logic in consensus to track and drop clients that exceed a number of failed tx proposals ([#3208], [#3155], [#3156])
```
Comment (samdealy) [inline CHANGELOG.md:16]:
```suggestion
- Add logic in consensus to track and drop clients that exceed a number of failed tx proposals ([#3208], [#3155], [#3156])
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -8,14 +8,264 @@ The crates in this repository do not adhere to [Semantic Versioning](https://sem
 
 ## [5.0.0]
 
+### Added
+
+- Average slam rate to json slam report ([#3080])
+- Provide build time error messages for invalid SGX_MODE and IAS_MODE environment variables ([#3164])
+- `mc-attest-verifier-config` crate to handle multiple enclave measurements and hardening advisories ([#3148])
```
Comment (samdealy) [inline CHANGELOG.md:15]:
```suggestion
- Use `mc-attest-verifier-config` crate to handle multiple enclave measurements and hardening advisories ([#3148])
```

**Review by samdealy — APPROVED** [review]
Overall looks good. I'd say just make the changelog descriptors full sentences because it seems to be the convention from earlier changelogs.

File: CHANGELOG.md (hunk 1 above)
Comment (nick-mobilecoin) [reply CHANGELOG.md]: reworded

File: CHANGELOG.md (hunk 2 above)
Comment (nick-mobilecoin) [reply CHANGELOG.md]: reworded

File: CHANGELOG.md (hunk 3 above)
Comment (nick-mobilecoin) [reply CHANGELOG.md]: reworded

File: CHANGELOG.md (hunk 5 above)
Comment (nick-mobilecoin) [reply CHANGELOG.md]: reworded

File: CHANGELOG.md (hunk 6 above)
Comment (nick-mobilecoin) [reply CHANGELOG.md]: reworded

File: CHANGELOG.md (hunk 7 above)
Comment (nick-mobilecoin) [reply CHANGELOG.md]: reworded

File: CHANGELOG.md (hunk 8 above)
Comment (nick-mobilecoin) [reply CHANGELOG.md]: reworded

File: CHANGELOG.md (hunk 4 above)
Comment (nick-mobilecoin) [reply CHANGELOG.md]: reworded

**Review by varsha888 — APPROVED** [review]

**Pull request merged**